src/mainBlock.css

```diff
@@ -0,0 +1,34 @@
+.stars{
+  display: flex;
+}
+
+.stars__star{
+  background-position: center;
+  background-repeat: no-repeat;
+  margin: 4px;
+  height: 16px;
+  width: 16px;
+  display: flex;
+  list-style: none;
+  background-image: url("images/star-active.svg");
+}
+
+.stars--0 .stars__star:nth-child(n) {
+  background-image: url("images/star.svg");
+}
+
+.stars--1 .stars__star:nth-child(n+2) {
+  background-image: url("images/star.svg");
+}
+
+.stars--2 .stars__star:nth-child(n+3) {
+  background-image: url("images/star.svg");
+}
+
+.stars--3 .stars__star:nth-child(n+4) {
+  background-image: url("images/star.svg");
+}
+
+.stars--4 .stars__star:nth-child(n+5) {
+  background-image: url("images/star.svg");
+}
```

Comment:
The CSS rules for `.stars--n .stars__star:nth-child(n+x)` are repetitive. You can optimize this by using a more efficient selector or a loop in a preprocessor like SASS to reduce redundancy. This will make your CSS more maintainable and cleaner.